plugins/@grouparoo/mongo/src/lib/query-import/executeQuery.ts

```diff
@@ -0,0 +1,8 @@
+import { ExecuteQueryMethod } from "@grouparoo/app-templates/dist/source/query";
+
+export const executeQuery: ExecuteQueryMethod = async ({
+  connection,
+  query,
+}) => {
+  return connection.execute(query);
```

Comment:
`connect` returns `{ client, db }` so isn't this `connection.db.query()` or something?
Is `client` actually used or does it already have a point to the `client` ? Just `db` might be better.